src/dotnet/commands/dotnet-new/FSharp_Console/project.json.template

```diff
@@ -16,8 +16,15 @@
   },
 
   "tools": {
-       "dotnet-compile-fsc": "1.0.0-*"
-   },
+    "dotnet-compile-fsc": {
+      "version": "1.0.0-*",
+      "imports": [
```

Comment:
These imports look super wrong. Why are they needed? Can you list out the nuget restore failures?